src/features/LoginForm/index.ts

```diff
@@ -0,0 +1,3 @@
+import { LoginForm } from './ui/LoginForm';
```

Comment:
Здесь и в других файлах можно экспортировать без прямого импорта
```suggestion
export { LoginForm } from './ui/LoginForm';
```

Reply:
изменила